frontend/lib/uast/post-parse-checks.cpp

```diff
@@ -601,10 +579,10 @@ void Visitor::checkFormalsForTypeOrParamProcs(const Function* node) {
 
     if (doEmitError) {
       CHPL_ASSERT(formalIntentStr);
-      error(decl, "cannot use '%s' intent in a function returning "
-                  "with '%s' intent.",
-                  formalIntentStr,
-                  returnIntentStr);
+      CHPL_POSTPARSE_ERR_SIMPLE(builder_, decl,
+                                "cannot use '" + std::string(formalIntentStr) +
+                                    "' intent in a function returning with '" +
```

Comment:
Perhaps extract the string into a constant so the line wrapping is less weird (it's correct/consistent here, just confusing).

Reply:
Per your other comment about the number of unwieldy string operations I'll come up with a more general solution to that which should also address this instance.